src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -46,6 +46,23 @@ extern "C" {
         DLOGP("[%s] Time taken: %" PRIu64 " ms", msg, (GETTIME() - (t)) / HUNDREDS_OF_NANOS_IN_A_MILLISECOND);                                       \
     } while (FALSE)
 
+#define PROFILE_CALL_WITH_START_END_T_OBJ(f, s, e, d, msg)                                                                                           \
+    do {                                                                                                                                             \
+        s = GETTIME() / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;                                                                                          \
```

Comment:
Let us not calculate MILLISECONDS here. These should only be timestamp recording. The millisecond conversion should happen in `d= e - s`. Also, just a nit. But let us be consistent with naming across the macros and switch to t1, t2, etc.

Reply:
That is because we need to send it across in milliseconds, I moved all the conversion outside of the macros to inside as per your previous comments

Reply:
Ok yeah. My bad then. We are sending start and end timestamps and that is why we are doing this.